CHANGES.md

```diff
@@ -29,10 +29,12 @@ creating a new release entry be sure to copy & paste the span tag with the
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
 -------------------------------------------------------------------------------
-## __cylc-8.0rc2 (<span actions:bind='release-date'>Upcoming</span>)__
+## __cylc-8.0rc2 (<span actions:bind='release-date'>Released 2022-03-23</span>)__
 
 Second Release Candidate for Cylc 8 suitable for acceptance testing.
 
+Note: Cylc 8 beta users will not be able to restart workflows run with previous Cylc 8 pre-releases due to changes in the workflow database structure.
+
```

Comment:
I not sure we need this note, it suggests you won't be able to restart 8.0rc1 workflows but you can. If users jumping from a beta version to rc2 want to see what's changed, they should also read the changelog for rc1 imo

Reply:
I added it because, due to the close timeframes of the releases, I thought users may find this useful information to have on hand if they pick up this release, skipping rc1. Happy to change the wording to pre-rc releases or remove if you feel strongly.

Reply:
Reverted.